examples/tina-cloud-starter/components/tina-wrapper.tsx

```diff
@@ -24,27 +22,13 @@ import { LoadingPage } from './Spinner'
  * if you're on a route that starts with "/admin"
  */
 const TinaWrapper = (props) => {
-  const cms = React.useMemo(() => {
-    return new TinaCMS({
-      apis: {
-        tina: createClient(),
-      },
-      sidebar: {
-        placeholder: SidebarPlaceholder,
-      },
-      enabled: true,
-    })
-  }, [])
-
-  /** Disables the TinaCMS "Media Manager" */
-  cms.plugins.all('screen').forEach((plugin) => {
-    if (plugin.name === 'Media Manager') {
-      cms.plugins.remove(plugin)
-    }
-  })
-
   return (
-    <TinaCloudAuthWall cms={cms}>
+    <TinaCloudAuthWall
```

Comment:
That being said, the one tricky part of this API is that if you specify a CMS object for any customization, the `clientID`  and `organization` passed to the component no longer do anything.  Probably not a huge deal, as we check for this type of thing at runtime, but I wonder if there's a possible refactor that would allow us to check this through Typescript.

Reply:
Hmm not sure the tricky part would be if knowing if localClient is true or not and they have union types based on that. If its in an environment variable typescript will have no way of knowing.

Reply:
Yeah, that might be tricky. Just left a note [here]( https://github.com/tinacms/tina-graphql-gateway/pull/254/files#r651076223) on a perhaps simpler (and non-Typescript dependant) solution.